ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-presubmits.yaml

```
@@ -463,6 +463,63 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )gke-scale-tests,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

```suggestion
    always_run: false
```
And use `skip_if_only_changed: ^ui/` https://docs.prow.k8s.io/docs/jobs/#triggering-jobs-based-on-changes

---

Added that in [fec3275](https://github.com/openshift/release/pull/40290/commits/fec32759fea1fdc1840a8e55a250542dc96f125c)